pkg/policy/api/l4.go

```diff
@@ -28,6 +28,12 @@ func (l4 L4Proto) IsAny() bool {
 	return l4 == ProtoAny || string(l4) == ""
 }
 
+// SupportedProtocols returns the currently supported protocols in the policy
+// engine, excluding "ANY".
+func SupportedProtocols() []L4Proto {
+	return []L4Proto{ProtoTCP, ProtoUDP, ProtoSCTP}
```

Comment:
Why don't we have `ICMP` nor `ICMPv6`?

Reply:
We do not support selecting ICMP as protocol that can be allowed or denied. [ICMP gets its own unique field](https://docs.cilium.io/en/latest/security/policy/language/#limit-icmp-icmpv6-types) so that users can't accidentally block discovery messages. Only these protocols are supported for network policy selection when using the general PortoProtocol type.